services/web/client/source/class/osparc/component/widget/NewFogbugzIssue.js

```diff
@@ -0,0 +1,64 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2020 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
+/**
+ * Here is a little example of how to use the widget.
+ *
+ * <pre class='javascript'>
+ *   const url = osparc.component.widget.NewFogbugzIssue.getNewIssueUrl();
+ *   window.open(url);
+ * </pre>
+ */
+
+qx.Class.define("osparc.component.widget.NewFogbugzIssue", {
```

Comment:
this is some static utility code, should it be under component or widget... inside the widget that uses it? or inside the utils class?

Reply:
Done